xbmc/guilib/GUIMoverControl.dox

```diff
@@ -37,10 +43,42 @@ important, as xml tags are case-sensitive.
 |----------------:|:--------------------------------------------------------------|
 | texturefocus    | Specifies the image file which should be displayed when the mover has focus. [See here for additional information about textures](http://kodi.wiki/view/Texture_Attributes).
 | texturenofocus  | Specifies the image file which should be displayed when the mover does not have focus.
+| movingspeed     | Specifies the allowed directional movements and their configuration. The <c>`<movingspeed>`</c> must contains as many <c>`<eventconfig>`</c> tags as required.
 
 
 --------------------------------------------------------------------------------
-\section Mover_Control_sect3 See also
+\section Resize_Control_sect3 Note on use of movingspeed tag
+
+The <c>`movingspeed`</c> tag must be specified to allow the control to be moved via
+an input device such as keyboard. Each direction can be specified and configured
+to simulate the motion effect with the <c>`eventconfig`</c> tag.
+
+The following attributes allow the motion effect of the control to be configured:
+
+| Attribute       | Description                                                   |
+|----------------:|:--------------------------------------------------------------|
+| type            | Specifies the direction of movement, accepted values are: <c>`up`</c>, <c>`down`</c>, <c>`left`</c>, <c>`right`</c>, all types of direction are optional.
+| acceleration    | Specifies the acceleration in pixels per second (integer value).
+| maxvelocity     | Specifies the max movement speed, it depends from acceleration value, a suitable value could be (acceleration value)*3. Set 0 to disable it. (integer value).
+| resettimeout    | Specifies the idle timeout to resets the acceleration speed in millisecs (integer value).
+| delta           | Specifies the minimal pixel increment step (integer value).
+
+The attributes <c>`acceleration`</c>, <c>`maxvelocity`</c>, <c>`resettimeout`</c>,
+<c>`delta`</c> must be specified individually for each <c>`eventconfig`</c> tag,
+or globally in to <c>`movingspeed`</c> tag, or also mixed, as the following example:
```

Comment:
or globally in the <c>`movingspeed`</c> tag, or a mixture, as the following example: